modules/cms/classes/Controller.php

```diff
@@ -1605,4 +1626,27 @@ protected function verifyCsrfToken()
             $token
         );
     }
+
+    /**
+     * Removes '@' from soft component name
+     * @param string $label
+     * @return string
+     */
+    protected function parseComponentLabel($label)
+    {
+        if ($this->isSoftComponent($label)) {
+            return str_replace('@', '', $label);
```

Comment:
@DanHarrin Could this potentially strip legitimate `@` symbols?

Reply:
@bennothommo potentially, I've replaces `str_replace()` with `ltrim()`.